examples/acl/src/backend/app/models/users.clj

```diff
@@ -0,0 +1,72 @@
+(ns models.users
+  (:require
+    [clojure.core :as core]
+    [honeysql.helpers :refer :all :as helpers]
+    [xiana.core :as xiana])
+  (:import
+    (java.util
+      UUID)))
+
+(defn ->store-user
+  [m]
+  (let [u (select-keys m [:id
+                          :password
+                          :last_login
+                          :is_superuser
+                          :username
+                          :first_name
+                          :last_name
+                          :email
+                          :is_staff
+                          :is_active])]
+    (cond-> u
```

Comment:
Do you want this to run a single time?

Reply:
> Do you want this to run a single time?

I don't like it too, looks like muuntaja doesn't converts booleans from json body. Needs more inspection.